README.md

```diff
@@ -132,31 +178,55 @@ The following tables lists the configurable parameters of the Harbor chart and t
 | `chartmuseum.image.tag` | Tag for chartmuseum image | `dev` |
 | `chartmuseum.replicas` | The replica count | `1` |
 | `chartmuseum.resources` | [resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/) to allocate for container   | undefined |
-| `chartmuseum.volumes` | used to create PVCs if persistence is enabled (see instructions in values.yaml) | see values.yaml |
 | `chartmuseum.nodeSelector` | Node labels for pod assignment | `{}` |
 | `chartmuseum.tolerations` | Tolerations for pod assignment | `[]` |
 | `chartmuseum.affinity` | Node/Pod affinities | `{}` |
-| **Storage For Registry And Chartmuseum** |
-| `storage.type` | The storage backend used for registry and chartmuseum: `filesystem`, `azure`, `gcs`, `s3`, `swift`, `oss` | `filesystem` |
-| `other values` | The other values please refer to https://github.com/docker/distribution/blob/master/docs/configuration.md#storage |  |
 | **Clair** |
-| `clair.enabled` | Enable Clair? | `true` |
+| `clair.enabled` | Enable Clair | `true` |
 | `clair.image.repository` | Repository for clair image | `goharbor/clair-photon` |
 | `clair.image.tag` | Tag for clair image | `dev`
 | `clair.replicas` | The replica count | `1` |
-| `clair.proxy.enabled` | Enable the use of http(s) proxy | `false` |
-| `clair.proxy.http_proxy` | URL of the http proxy |
-| `clair.proxy.https_proxy` | URL of the https proxy |
+| `clair.httpProxy` | URL of the http proxy ||
```

Comment:
add `URL of the http proxy, via which Clair will download the vulnerability data from the internet`